streamer/periodconcat_node.py

```diff
@@ -130,9 +168,25 @@ def find(elem: ElementTree.Element, *args: str) -> ElementTree.Element:
       # but it won't allow putting comments at the begining of the file.
       contents += ElementTree.tostring(element=concat_mpd, encoding='unicode')
       master_dash.write(contents)
-      
+  
   def _hls_concat(self) -> None:
-    """Concatenates multiple HLS playlists with #EXT-X-DISCONTINUITY."""
+    """Concatenates multiple HLS playlists using #EXT-X-DISCONTINUITY."""
+    
+    # Initialize the HLS concater with a sample Master HLS playlist and
+    # the output direcotry of the concatenated playlists.
+    hls_concater = HLSConcater(os.path.join(self._packager_nodes[0].output_dir,
```

Comment:
It took me a minute to read and understand this join.  Can you make it easier to read by assigning this path to a local variable with an explanatory name?  Perhaps something like first_playlist_path?